e2e/src/helpers.ts

```diff
@@ -14,6 +14,22 @@ export function log(...args: any[]) {
 // a collection of useful playwright actions
 // these could evolve into fixtures: https://playwright.dev/docs/test-fixtures
 
+export function debugPageConsole(page: Page) {
+  page.on("console", (msg) => console.log(msg.text()));
+}
```

Comment:
This is a handy function for printing editor console messages in a playwright run for debugging.